src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -413,10 +416,76 @@ private void Initialize()
 
             Focusable = true;
             KeyEvent += OnKeyEvent;
+
+            Accessibility.Accessibility.ScreenReaderEnabled += (s, e) => {
+                isScreenReaderEnabled = true;
+                if (loopEnabled) ShowItemsForAccessibility(currentValue - middleItemIdx);
+                else
+                {
+                    //Exception case handling condition state.
+                    //If user sets 4 items it can scroll but not loop.
+                    if (currentValue > (middleItemIdx * 2)) ShowItemsForAccessibility(middleItemIdx + 1);
+                    else ShowItemsForAccessibility(middleItemIdx);
+                }
+            };
+            Accessibility.Accessibility.ScreenReaderDisabled += (s, e) => {
+                isScreenReaderEnabled = false;
+                HideItemsForAccessibility();
+                Accessibility.Accessibility.ClearCurrentlyHighlightedView();
```

Comment:
You do not have call ```ClearCurrentlyHighlightedView```
The screen-reader will clear highlight.

Reply:
I will remove this. but i think the hilight is not gone without this API calling.